tsfile/src/main/java/org/apache/iotdb/tsfile/compress/ICompressor.java

```diff
@@ -52,6 +52,8 @@ static ICompressor getCompressor(CompressionType name) {
         return new NoCompressor();
       case SNAPPY:
         return new SnappyCompressor();
+      case LZ4:
+        return new IOTDBLZ4Compressor();
```

Comment:
To be consistent with LZ4UnCompressor and simplified, rename this to LZ4Compressor?

Reply:
The package I import is called LZ4Compressor, so I can't use this name